src/server/debug.rs

```diff
@@ -360,6 +371,97 @@ impl Debugger {
         Ok(())
     }
 
+    pub fn recover_all(&self, threads: usize, read_only: bool) -> Result<()> {
```

Comment:
Please add a test for this function. Load some corrupted data into local RocksDB and recover it.

Reply:
The recovering logic is already tested in `test_mvcc_checker` in debug.rs